packages/foam-vscode/package.json

```diff
@@ -261,7 +261,7 @@
           "default": true
         },
         "foam.decorations.links.enable": {
-          "description": "Enable decorations for links",
+          "description": "Enable decorations for placeholders",
```

Comment:
I feel inclined to remove this setting.

Reply:
Actually, I am pretty happy to remove it as well.
If people loudly complain about it, and have good reasons to do so, we might reintroduce it

Reply:
Done.